tests/php/test-fieldmanager-colorpicker-field.php

```diff
@@ -27,6 +27,15 @@ public function test_basic_render() {
 		$this->assertRegExp( '#<input class="[^"]*fm-colorpicker-popup[^>]+name="test_colorpicker"#', $html );
 	}
 
+	public function test_default_color() {
+		$fm = new Fieldmanager_Colorpicker( array( 'name' => 'test_colorpicker', 'default_value' => '#ff0000' ) );
+
+		ob_start();
+		$fm->add_meta_box( 'Test Colorpicker', 'post' )->render_meta_box( $this->post, array() );
+		$html = ob_get_clean();
+		$this->assertEquals( 1, preg_match( '[data-default-color="#ff0000"]', $html ) );
```

Comment:
while technically valid, square brackets are pretty uncommon for regex delimiters, especially since they're special characters in regex. The typically delimiter is `/`, except when that character is used in the regex, then you'll commonly find `#` used.

All that said, make this `'/data-default-color="#ff0000"/'`